docs/releasing.md

```diff
@@ -24,6 +24,13 @@ Creating a federation v2 release involves the following steps:
    3. `kubefed2 version` (check that the output is as expected)
    4. `tar cvzf kubefed2.tar.gz kubefed2`
    5. `sha256sum kubefed2.tar.gz > kubefed2.tar.gz.sha`
+4. Package the helm chart for release
+   1.  Adjust the default image tag in values.yaml
+   2.  Update the chart version in Chart.yaml
+   3.  Package the chart using `helm package federation-v2`
+   4.  Place the release packaged chart (.tgz file) in the federation-v2/ folder.
+   5.  Add the new version to the chart index `helm repo index federation-v2/ --url=https://raw.githubusercontent.com/kubernetes-sigs/federation-v2/master/charts/federation-v2/`
+   6.  Ensure index.yaml containes the latest release within the entries array.
```

Comment:
Once the version has been added to the chart index, it will be necessary to revert the changes from steps 1 and 2 (they should never be committed to master) and create a PR that captures the changes to index.yaml.

Reply:
Adjusted. `   7.  Create PR that captures index.yaml for branch "add-chart-repo-index"` Acceptable?